app/Http/Controllers/Admin/AircraftController.php

```diff
@@ -78,7 +78,7 @@ public function index(Request $request)
     public function create(Request $request)
     {
         return view('admin.aircraft.create', [
-            'airports'    => $this->airportRepo->selectBoxList(),
```

Comment:
Commas have to have space after them. That could be a reason it's failing the build/styleci